hail/src/test/scala/is/hail/TestUtils.scala

```diff
@@ -414,7 +414,14 @@ object TestUtils {
     val t = x.typ
     assert(t.typeCheck(expected), t)
 
-    ExecStrategy.values.intersect(execStrats).foreach { strat =>
+    val filteredExecStrats: Set[ExecStrategy] =
+      if (HailContext.get.backend.isInstanceOf[SparkBackend]) ExecStrategy.values.intersect(execStrats)
```

Comment:
I think BackEnd should have a `supportedStrategies` field, and then you can intersect `execStrats` with that.

Isn't ExecStrategy.values all the values?  So this intersection does nothing?

Also: if formatting.